scripts/eurostat/health_determinants/alcohol_consumption/process.py

```diff
@@ -0,0 +1,820 @@
+"""
+from sys import path
+# For import common.replacement_functions
+# path.insert(1, '../')
+# from common.replacement_functions import (_replace_sex, _replace_frequenc,
+#                                           _replace_isced11, _replace_quant_inc,
+#                                           _replace_deg_urb, _replace_c_birth,
+#                                           _replace_citizen, _split_column)
+# For import util.alpha2_to_dcid
+path.insert(1, '../../../../')
+
+import os
+import pandas as pd
+import numpy as np
+from util.alpha2_to_dcid import COUNTRY_MAP
+from absl import app
+from absl import flags
+
+_FLAGS = flags.FLAGS
+default_input_path = os.path.dirname(
+    os.path.abspath(__file__)) + os.sep + "input_files"
+flags.DEFINE_string("input_path", default_input_path, "Import Data File's List")
+
+#alcoholconsumption_by_sex_educationattainment
+def alcoholconsumption_by_sex_education(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Cleans the file alcoholconsumption_by_sex_education
+    for concatenation in Final CSV
+    Args:
+        df (pd.DataFrame): The raw df as the input
+    Returns:
+        df (pd.DataFrame): provides the cleaned df as output
+    """
+    # providing column names
+    columns = ['unit,frequenc,isced11,sex,age,geo', '2019', '2014']
+    df.columns=columns
+    # spliting the first column to multiple columns
+    split_columns = "unit,frequenc,isced11,sex,age,geo"
+    df = _split_column(df,split_columns)
+    # Filtering out the wanted rows and columns    
+    df = df[df['age'] == 'TOTAL']
+    df = df[(df['geo'] != 'EU27_2020') & (df['geo'] != 'EU28')]
+    df = _replace_frequenc(df)
+    df = _replace_sex(df)
+    df = _replace_isced11(df)
+    # giving proper statvar name
+    df['SV'] = 'Percent_AlcoholConsumption_'+\
+        df['frequenc']+'_In_Count_Person_'+\
+            df['isced11']+'_'+df['sex']
+    # dropping unwanted columns
+    df.drop(columns=['unit','age','isced11','frequenc','sex'],inplace=True)
+    # ask
+    df = df.melt(id_vars=['SV','geo'], var_name='time'\
+            ,value_name='observation')
+    return df
+
+#alcoholconsumption_by_sex_income
+def alcoholconsumption_by_sex_income(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Cleans the file alcoholconsumption_by_sex_income for concatenation in Final CSV
+    Args:
+        df (pd.DataFrame): The raw df as the input
+    Returns:
+        df (pd.DataFrame): provides the cleaned df as output
+    """
+    # providing column names
+    columns = ['unit,quant_inc,frequenc,sex,age,geo', '2019', '2014']
+    df.columns=columns
+    # spliting the first column to multiple columns
+    split_columns = "unit,quant_inc,frequenc,sex,age,geo"
+    df = _split_column(df,split_columns)
+    # Filtering out the wanted rows and columns
+    df = df[df['age'] == 'TOTAL']
+    df = df[(df['geo'] != 'EU27_2020') & (df['geo'] != 'EU28')]    
+    df = _replace_frequenc(df)
+    df = _replace_sex(df)
```

Comment:
As a best practise define function before it is used. E.g., in this case move function before it is called (i.e., above alcoholconsumption_by_sex_income)